src/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs

```diff
@@ -204,5 +206,29 @@ private static SafeSecKeyRefHandle ImportKey(ECParameters parameters)
 
             return Interop.AppleCrypto.ImportEphemeralKey(blob, isPrivateKey);
         }
+
+        public unsafe int ImportSubjectPublicKeyInfo(
+            ReadOnlySpan<byte> source,
+            out int bytesRead)
+        {
+            fixed (byte* ptr = &MemoryMarshal.GetReference(source))
+            {
+                using (MemoryManager<byte> manager = new PointerMemoryManager<byte>(ptr, source.Length))
+                {
+                    // Validate the DER value and get the number of bytes.
+                    EccKeyFormatHelper.ReadSubjectPublicKeyInfo(
+                        manager.Memory,
+                        out int localRead);
+
+                    SafeSecKeyRefHandle publicKey = Interop.AppleCrypto.ImportEphemeralKey(source.Slice(0, localRead).ToArray(), false);
```

Comment:
Rather than add _n_ new places calling ToArray, perhaps change ImportEphemeralKey to take a ReadOnlySpan\<byte\>.  Even if it currently calls ToArray (rather than also plumbing the change through CFData) that reduces the maintenance burden later.

Reply:
(The least impactful version would be to make an overload that calls the existing one with ToArray(), so we don't end up double ToArray()ing on existing callers)